src/lib/ext-popup.js

```diff
@@ -44,46 +45,34 @@ export default class Popup {
       : this.renderMonitorStoppedUI();
   }
 
-  handleViewActivityLog() {
-    openActivityLogPage();
+  async handleViewActivityLog() {
+    await openActivityLogPage();
     window.close();
   }
 
   async handleEvent(event) {
     try {
-      if (event.type === 'click') {
```

Comment:
Removed this condition since we are have only attached [click event on the buttons](https://github.com/mozilla/extension-activity-monitor/blob/master/src/lib/ext-popup.js#L91-L93). So, the else call is unreachable in real scenario.

Reply:
To ensure that this continues the case, consider adding an assertion check with
https://developer.mozilla.org/en-US/docs/Web/API/Console/assert

Reply:
We have [enabled an Eslint rule to show error for console messages](https://github.com/mozilla/extension-activity-monitor/blob/master/.eslintrc.json#L9). Should we remove this?